internal/handlers/common/aggregations/stages/projection/projection_iterator.go

```diff
@@ -59,8 +59,9 @@ func (iter *projectionIterator) Next() (struct{}, *types.Document, error) {
 	}
 
 	projected, err := ProjectDocument(doc, iter.projection, iter.inclusion)
-	if err != nil {
-		return unused, nil, lazyerrors.Error(err)
+
+	if err = processOperatorError(err); err != nil {
```

Comment:
This line is not needed, it's unreachable because `ProjectDocument` already returns command error. :hugs:

Reply:
nice catch! Fixed